### ISSUE TITLE
Cannot quit/kill/uninstall when menubar icon is hidden

### ISSUE THREAD

**Issue by tborychowski** [Ji4n1ng/OpenInTerminal#109]
## What is the issue?
The app cannot be completely closed. Even from the Process Monitor - tried "Force quit" and it somehow restarts!
Because of that it's impossible to uninstall it.

## How does one reproduce the issue?
Try closing/quitting it.

## Screenshots
irrelevant 

## Debug information (Logs, OS, Hardware, Version, etc.)
MacOS Big Sur

**Comment by Ji4n1ng** [comment]
Finder toolbar extension and options in context menu will not disappear even if you quit the app. That's how the Finder extension works. It's not just OpenInTerminal, but those applications that have Finder extensions.

**Comment by tborychowski** [comment]
No, you misread the issue.
The title says: "Cannot quit/kill/uninstall when menubar icon is hidden".
What that means is the app runs in background. 
When I try to uninstall it (i.e. remove the app from `/Applications`) there's a warning: cannot remove the app because it's running.
I opened "Activity Monitor" and tried to kill it and it didn't work!
I had to unselect the "Launch at login" option and reboot to be able to quit it.
Than I've noticed (after a reinstall) that when it has the menubar icon - there's a quit option there.
Hence the issue title: "Cannot quit/kill/uninstall when menubar icon is hidden"
